man/systemd.service.xml

```diff
@@ -1157,6 +1157,35 @@
         cgroup associated with the service.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>OpenFile=</varname></term>
+        <listitem><para>Takes an argument of the form <literal>path:fd-name:options</literal>,
+        where:
+        <itemizedlist>
+            <listitem><simpara><literal>path</literal> is a path to a file or an <constant>AF_UNIX</constant> socket;</simpara></listitem>
+            <listitem><simpara><literal>fd-name</literal> is a name that will be associated with the file descriptor;
+            the name may contain any ASCII character, but must exclude control characters and ":", and must be at most 255 characters in length;
+            it is optional and, if not provided, defaults to the file name;</simpara></listitem>
+            <listitem><simpara><literal>options</literal> is a comma-separated list of access options;
+            possible values are
+            <literal>ro</literal> (read-only),
+            <literal>append</literal>,
+            <literal>truncate</literal>,
+            <literal>ignore</literal>;
```

Comment:
Thinking a bit more, "ignore" isn't a great name for this option, let's rename it to "graceful" as Lennart suggested.